modules/structs/issue.go

```diff
@@ -26,8 +26,9 @@ const (
 
 // PullRequestMeta PR info if an issue is a PR
 type PullRequestMeta struct {
-	HasMerged bool       `json:"merged"`
-	Merged    *time.Time `json:"merged_at"`
+	HasMerged        bool       `json:"merged"`
+	Merged           *time.Time `json:"merged_at"`
+	IsWorkInProgress bool       `json:"wip"`
```

Comment:
I am against this naming as we intend to be as GitHub API-compatible as possible, unless there is a good reason to deviate.
In the GitHub API, this field is called `draft`

Reply:
Can you post the API URL?

Reply:
afaik we only use `Draft` for releases.
Generally we refer to it as `WIP` in other code/docs for PRs.

Reply:
https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28#list-pull-requests
Navigate to `response schema` and search for `draft`

Reply:
@delvh did @jolheiser's edit change your mind? 

If not, I propose I rename `wip` to `draft` to stay consistent with GitHub's naming, and leave `IsWorkInProgress` as is to stay consistent with the rest of the codebase.

Reply:
Hmm… Difficult question:
Do we want to be GitHub API compliant, or do we want to follow our own terminology?
I think it is better if we stay compliant.
Especially now that we have Actions, every deviation will be noticeable in actions that are written to talk with GitHubs API…

Reply:
Or maybe we can have both fields and check wip first and then draft.

Reply:
I'm not sure if I understand the second part. Are `wip` and `draft` not synonymous in this case? Why would you check both?

Reply:
I'm fine with it remaining compatible. I think having duplicate fields would just be overcomplicated, so `draft` works for me. :+1: